modules/govuk_python3/manifests/init.pp

```diff
@@ -0,0 +1,23 @@
+# == Class: govuk_python3
+#
+# Install python3 package
+#
+class govuk_python3 (
+  $govuk_python_version = '3.6.12',
+  $apt_mirror_hostname,
+  $apt_mirror_gpg_key_fingerprint,
+) {
+
+  apt::source { 'govuk-python3':
+    location     => "http://${apt_mirror_hostname}/govuk-python",
+    release      => 'stable',
+    architecture => $::architecture,
+    repos        => 'main',
+    key          => $apt_mirror_gpg_key_fingerprint,
+  }
+
+  package { 'govuk-python3':
+    ensure  => $govuk_python_version,
+    require => Apt::Source['govuk-python'],
```

Comment:
Should this be:

```suggestion
    require => Apt::Source['govuk-python3'],
```

?

Reply:
No, the source is still govuk-python, its just the version number thats different

Reply:
hmmm, actually, looks like the branch is failing now, I'll try with the change